nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -370,7 +393,13 @@ def filter_func(point: StatisticPoint) -> bool:
             return MinMaxQuantization in point.algorithm_to_tensor_collectors and \
                    point.target_point.type == quantization_target_point.type
 
+        unified_ops_list = []
+        for unified_ops_group in self._unified_scale_groups:
```

Comment:
I would prefer to see implicit calculation `self._unified_scale_groups` instead of explicit calculation it into `self._get_quantization_target_points(model)`. Something like that:

```
1. quantizer_setup = self._get_quantizer_setup(nncf_graph, pattern)
2. unified_scale_groups = self._collect_unified_groups(quantizer_setup)
3. quantization_target_points = self._get_quantization_target_points(quantizer_setup, nncf_graph)
```